core-bundle/contao/languages/en/tl_page.xlf

```diff
@@ -320,6 +320,12 @@
       <trans-unit id="tl_page.hide.1">
         <source>Hide the page in the navigation menu.</source>
       </trans-unit>
+      <trans-unit id="tl_page.guests.0">
+        <source>Hide in navigation if logged in</source>
```

Comment:
```suggestion
        <source>Hide in navigations if logged in</source>
```

Reply:
I don't think the plural is correct here.

Reply:
We also don't use the plural in `tl_page.hide`.